metrics/lib/mem-load.md

```diff
@@ -0,0 +1,52 @@
+# `mem-load` stack stresser
```

Comment:
We should ref this doc from the top level doc, like we do for the cpu-load: https://github.com/clearlinux/cloud-native-setup/tree/master/metrics#cpu-load-generator